### DISCUSSION TITLE
Theme Question

### DISCUSSION THREAD

**Discussion by tyeeman** [jaredkrinke/md2blog#27] [General]
This is really great. Thanks for writing it. Fuctionally, it works perfect. Only suggestion I can add is I need to use my own theme. Could you add the use of an external html file where I could just add a "content" tag somewhere in it. I guess I would need a couple of others for "title' and "topics" also.

**Comment by jaredkrinke** [comment]
Not sure how I missed this question, sorry! My suggestion would be to simply fork md2blog and then play around with the built-in templates ("templates.ts").

**Comment by c-txt** [comment]
This is such a great and useful tool that works awesomely for my needs and i couldn't find anything simpler for a non-technical person like me. My only issue is that the generated html files have a dark theme which works fine for developers i guess but not for writers like me. Would it be possible to provide a light theme as well?
